docs/cover-video.md

```diff
@@ -0,0 +1,40 @@
+# Updating featured cover video
+
+Atlas will fetch a remote `cover-info.json` file to get information on what should be displayed in the hero section on top of the home page. That JSON file is hosted on our Linode object storage, and a designated community member will receive write permissions to that storage, so they can update the content dynamically.
+
+
+### Structure of the JSON file
+
+```json
+{
+  "videoId": "1",
+  "coverTitle": "Ghost Signals",
+  "coverDescription": "How We Lost Trust In Authority, And Authority Taught Us To Distrust Ourselves",
+  "coverCutMediaUrl": "https://eu-central-1.linodeobjects.com/atlas-hero/cover-cut-ghost-signals.mp4"
+}
+```
+
+- `videoId`: ID of the video uploaded to Joystream that this cover mentions
+- `coverTitle`: Title that should be displayed in the cover section. In general, this should be the same at the title of the published video, but may need to be shortened for some videos for the cover to look sexy
+- `coverDescription`: Description that should be displayed under the title in the cover section. Not required but in that case an empty string should be provided. (`"coverDescription": ""`)
+- `coverCutMediaUrl`: URL to the video file that should be playing in the background of the cover section. Usually this will be a trimmed version of the published video. This can be hosted in the same Linode bucket as `cover-info.json`
```

Comment:
```suggestion
- `coverCutMediaUrl`: URL to the video file that should be playing in the background of the cover section. Usually, this will be a trimmed version of the published video. This can be hosted in the same Linode bucket as `cover-info.json`
```